### PR TITLE
Added a #!/bin/bash to the top of the clang format script.

### PR DESCRIPTION
**Pull request by calewis**
Attempted fix for #671 just FYI running the script generates failures on on current devel.

### CODE REVIEW

**Comment by feldergast** [comment]
What platform are you running clang-format on?  And which version?  Devel runs clean using clang-format 12 on my Mac and on sst-devel (which is Linux, but I'm not sure what version).

**Comment by calewis** [comment]
Okay this is going to be awkward I'm using clang-format13. I suspect that is the issue.

**Comment by feldergast** [comment]
Yea, we were warned to pick a version and stick with it because they make non-backward compatible changes between versions.  We ended up going with version 12.

**Review by allevin — DISMISSED** [review]
Looks good to me

**Comment by feldergast** [comment]
Did clang-format-test.sh get added to the Makefile for make dist?  If not, we should probably add it.

**Comment by allevin** [comment]
> Did clang-format-test.sh get added to the Makefile for make dist? If not, we should probably add it.

Good catch!   No I did not add it... @calewis can you add this to your fork?

**Comment by calewis** [comment]
Sure just tell me where I need to add it to make it go.

**Comment by feldergast** [comment]
It goes in the EXTRA_DIST section near the top of Makefile.am.

**Comment by nmhamster** [comment]
@feldergast / @calewis - setting this to WIP to prevent autotester clogging. I think we are waiting on the addition of the script in `Makefile.am` right?

**Comment by calewis** [comment]
@nmhamster yeah I just added it to the first `Makefile.am` above the dir it was in.  If it needs to go somewhere else just let me know.

**Comment by nmhamster** [comment]
@calewis - I removed the WIP, let's see what the build says.

File: Makefile.am [hunk 1]
```diff
@@ -12,6 +12,7 @@ sstconf_DATA = src/sst/sstsimulator.conf
 EXTRA_DIST = \
 	autogen.sh \
 	CONTRIBUTING.md \
+	scrips/clang-format-test.sh \
```
Comment (nmhamster) [inline Makefile.am:15]: Should this be `scripts` not `scrips` ?

Comment (calewis) [reply Makefile.am:15]: 🤷‍♀️ yeah probably

Comment (nmhamster) [reply Makefile.am]: Have you developed love or hatred for the auto-tester yet ? :)

Comment (calewis) [reply Makefile.am]: I just try to ignore it :)

**Review by nmhamster — APPROVED** [review]

**Pull request merged**